content/03.methods.md

```diff
@@ -70,23 +70,24 @@ A summary of processed gender predictions of quotes at each point of processing
-humaniformat identifies if names are reversed (Lastname, Firstname), as well as identifies middle names.
+humaniformat is a rule based program that uses character markers to identify if names are reversed (Lastname, Firstname), find middle names and titles.
 This processing was not required for quote prediction because names written in news articles did not appear to be reversed or abbreviated.
 Since many last or first authorships may be non-names, we additionally filtered out any identified names if they partially or fully match any of the following terms: "consortium", "group", "initiative", "team", "collab", "committee", "center", "program", "author", or "institute".
 Furthermore, since many papers only contain first name initials (for example, "N. Davidson"), we remove any names less than four letters (length includes punctuation) and containing a "." or "-", then strip out all periods from the first name.
 This ensures that hyphenated names are not changed, e.g. Julia-Louise remains unchanged, but removes hyphenated initials, e.g. J-L.
-Finally, we only consider any remaining first names of more than two characters.
-This is to eliminate first and middle jointly-initialized names.
-For example, "NR Davidson" would be reduced to "Davidson" and then eliminated due to the lack of a first name.
+Finally, we only consider any remaining first names of one or more characters.
+<!-- This is to eliminate first and middle jointly-initialized names.
+For example, "NR Davidson" would be reduced to "Davidson" and then eliminated due to the lack of a first name. -->
 A summary of processed author gender predictions at each point of processing is provided in Tables {@tbl:table2} - {@tbl:table4}.
 
 
 #### Name Formatting for Name Origin Prediction
 In contrast to the gender prediction, we require the entire name in all steps of name origin prediction.
 For names identified in the _Nature_ news articles, we use the same process as described for the gender prediction; we again try to identify the full name.
 For author names, we process the names as previously described for the gender prediction of authors.
-For all names, we only consider them in our analyses if they consist of two distinct parts separated by a space.
-Additionally, if a full name is less than three characters, we were unable to consider it as the prediction model that we apply uses 3-mers.
+For all names, we only consider them in our analyses if they consist of two distinct parts separated by a space and excluding titles (e.g. Mrs., Prof.,  Dr., etc.).
+All names that were filtered out in the analysis of quotes and mentions are provided on our github in the file "data/author_data/all_mentioned_fullname_excluded.tsv" and "data/author_data/all_speaker_fullname_excluded.tsv".
+<!-- Additionally, if a full name is less than three characters, we were unable to consider it as the prediction model that we apply uses 3-mers. ## removed this because this does not occur. -->
```

Comment:
I did not re-train it. I only removed the need for the first and last name to be more than 3 characters. I ended up not retraining because minimum full-name size is 3 characters, 1 character for the first name, one for the last, and a space.